aip/0217.md

```diff
@@ -90,7 +81,7 @@ These facts lead to the following guidance:
     pages with unavailable resources or locations.
   - If a location or resource was unreachable during pagination, but the
     service tried again and it became reachable and the data was provided to
-    the user on a later page, the final page's `unreachable` field **must not**
+    the user on a later page, the final page's `unreachable` field **must**
```

Comment:
This sounds like the opposite of what we said in the call. I thought the idea was that we'd never, ever say that something was unreachable if we'd actually provided the resource.

Reply:
I think perhaps we misunderstood each other then. The goal of "unreachable" is to say "there are things that you should go look at... please go look at these things in more detail". 

That means that if at any point during the request a parent is unavailable, that parent should be listed in the "unreachable" list at the end of the listing. This means that even if it comes back online and the list *does* actually have all the items in it, we still want to tell folks "some funny stuff was going on with this thing. If you want to be fully confident that you have all the data, you should go investigate it again." 

If we exclude it, we're pretending that parent A never had any problems even though it's possible that there were some wonky things going on. 

If this is saying "If you ultimately provided a complete listing, then you shouldn't include the parent in `unreachable`, then I'm fine with "**must not**" as long as we change this to say "and the complete set of data was provided".

Reply:
My understanding was closer to @jskeet's, but I think that "and the complete set of data was provided" is valuable to add.